components/SelectorBase/SelectorBase.vue

```diff
@@ -247,6 +290,117 @@ export default {
     handleDelete() {
       this.selectedOption = '';
     },
+    removeElements(page) {
+      try {
+        page.getElementById('toolbar-administration').remove();
+        page
+          .getElementsByClassName('gin-secondary-toolbar layout-container')[0]
+          .remove();
+        page.getElementsByTagName('header')[0].style.top = 0;
+        page.getElementsByClassName('gin--vertical-toolbar')[0].className = '';
+        page
+          .getElementById('block-farm-powered')
+          .getElementsByTagName('a')[0]
+          .removeAttribute('href');
+      } catch (e) {
+        // couldn't remove elements
+      }
+    },
+    disableBackgroundElements(disable) {
+      // get the body of the page and remove/restore it from the tab index
+      // and hide/show it for screen readers
+      const element = document.getElementsByTagName('body')[0];
+      if (element) {
+        this.setAttributes(element, disable);
+      }
+    },
+    setAttributes(element, disable) {
```

Comment:
This seems sensible but I still seem to be tabbing through the elements in the hidden page.  If I click in the date and begin tabbing, eventually the URL of other elements appears at the bottom of the window when I continue to tab.  

Is that the same behavior your are experiencing?

Reply:
For me, the only tab-able elements are in the search bar (new tab, back button, bookmark etc.) and the background html tag element is also tab able. None of the background components or links were tab-able. I checked this by removing the code that re-enables the elements and closing the popup and seeing what else i could tab to.